queries/luadoc/highlights.scm

```diff
@@ -0,0 +1,133 @@
+; Keywords
+
+[
+  "@class"
+  "@type"
+  "@param"
+  "@alias"
+  "@field"
+  "@generic"
+  "@vararg"
+  "@diagnostic"
+  "@deprecated"
+  "@cast"
+  "@overload"
+  "@enum"
+  "@language"
+  "@see"
+  "extends"
+  (diagnostic_identifier)
+] @keyword
+
+[
+  "@async"
+] @keyword.coroutine
+
+(language_injection "@language" (identifier) @keyword)
```

Comment:
Shouldn't the identifier be something else?

Reply:
I couldn't think of a better fit, but yeah I agree. Here's an example

```lua
---@language JSON
local jsonText = [[{
    "name":"Emmy"
}]]
```

Reply:
Maybe `@constant`?

Reply:
Imo that doesn't feel exactly right, but is ok (another one could be `@type` but that also feels not exactly right), I'll just go w/ that

Reply:
Could just use `@parameter`.

Reply:
Yeah, that would fit better and be more consistent in highlighting with other annotations, that didn't cross my mind - thanks

Reply:
Forgot to push - fixed now